ghost/core/test/unit/api/cache-invalidation.test.js

```diff
@@ -0,0 +1,42 @@
+const assert = require('assert');
+const path = require('path');
+
+const glob = require('glob');
+
+const models = require('../../../core/server/models');
+const settings = require('../../../core/server/services/settings/settings-service');
+
+describe('Cache Invalidation', function () {
+    before(async function () {
+        // Initialise models - Utilised by various endpoints to reference static fields (i.e models.Post.allowedFormats) when required in
+        models.init();
+
+        // Initialise settings cache - Utilised by the indentites endpoint when required in
+        await settings.init();
+    });
+
+    it('Controller actions explicitly declare cacheInvalidate header', async function () {
+        const endpointPaths = glob.sync('*.js', {
```

Comment:
Would be good to have an extra assertion checking if there are any endpoint paths in case we move where our controllers live